spec/features/admin/order_cycles_spec.rb

```diff
@@ -113,6 +113,20 @@
     page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc2.id}"
     page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc3.id}"
 
+    # I can also clear filters using the 'clear all' button
+    select2_select schedule1.name, from: "schedule_filter"
+    select2_select oc1.suppliers.first.name, from: "involving_filter"
+    fill_in "query", with: oc0.name
+    page.should have_no_selector "#listing_order_cycles tr.order-cycle-#{oc0.id}"
+    page.should have_no_selector "#listing_order_cycles tr.order-cycle-#{oc1.id}"
+    page.should have_no_selector "#listing_order_cycles tr.order-cycle-#{oc2.id}"
+    page.should have_no_selector "#listing_order_cycles tr.order-cycle-#{oc3.id}"
+    click_on "Clear All"
+    page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc0.id}"
+    page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc1.id}"
+    page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc2.id}"
+    page.should have_selector "#listing_order_cycles tr.order-cycle-#{oc3.id}"
+
```

Comment:
Should we take advantage of working on this to change to RSpec's new syntax (`expect(page).to have(stuff)`) ?